src/scheduler/scheduler.html

```diff
@@ -0,0 +1,110 @@
+<h4 class="frame-title">{{'scheduler' | translate}}</h4>
+<motech-list paginated>
+    <motech-list-header>
+        <div class="button-group-primary">
+            <button class="button-add" ng-click="newJob()" ui-sref="scheduler.createOrUpdateJob({ action: 'new' })" type="button">{{'scheduler.scheduleJob' | translate}}</button>
+        </div>
+        <div class="button-group">
+            <button class="button-filter" type="button" uib-btn-checkbox ng-model="viewFilters" ng-click="showFilters()">{{'scheduler.filters' | translate}}</button>
```

Comment:
Ok, thanks for the note! I must have forgotten it... I will look at this once again